microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/get-started.md

```diff
@@ -5,341 +5,21 @@ author: MSEdgeTeam
-
-   Or, paste the github.io server URL, [https://microsoftedge.github.io/Demos/demo-to-do/](https://microsoftedge.github.io/Demos/demo-to-do/).
-
-1. In the demo app, enter a task, such as **test**.
-
-1. In the **Elements** tool, in the **Styles** tab, change a CSS value, such as: click the point size in `body { font-size: 11pt;}` and then change the value.  The corresponding `.css` file in the folder that you opened opens and it is automatically edited to match your changes made in the **Styles** tab (but is not saved).
-
-   ![Launch Instance > filepath > non-debug mode](./get-started-images/launch-instance-filepath-non-debug-mode.png)
-
-
-   **Close DevTools:**
-
-1. Click **Close** (**x**) in the **Edge DevTools** tab and in the **Edge DevTools: Browser** tab, if these tabs are open.
-
-*  Select Activity Bar > **Microsoft Edge Tools**.  If the **Targets** section lists any targets, hover on the right side of the target instance and then click **Close instance** (**x**).  The **Launch Instance** button appears, indicating that all DevTools instances are closed.
-
-1. Close `index.html` without saving changes.
-
-1. Select **File** > **Close Folder**.
-
-
-<!-- ====================================================================== -->
-## Step 4: Start DevTools by right-clicking an HTML file
-
-<!-- the equivalent section in Opening is more general than this section, which uses Demos repo -->
-
-Right-clicking an `.html` file in Visual Studio Code's **Explorer** is the main way to open DevTools when your webpage doesn't require running on a web server.
-
-*  Unlike the **Launch Instance** button, this approach opens DevTools in Debug mode.
-
-*  Unlike the **Launch Project** button that we'll use later, this approach doesn't require you to generate a `launch.json` file.
-
-
-We'll demonstrate by opening the **demo-to-do** web app:
-
-1. In Visual Studio Code, select **File** > **Open Folder**.
-
-1. Go to the directory where you cloned the Demos repo, open the specific directory for the **demo-to-do** app, such as `C:\Users\myusername\Documents\GitHub\Demos\demo-to-do\`, and then click the **Select Folder** button:
-
-   ![Open Folder: demo-to-do](./get-started-images/open-folder-demo-todo.png)
-
-1. Select **Activity Bar** > **Explorer** (![Explorer icon](./get-started-images/explorer-icon.png)) > right-click `index.html` and then select **Open with Edge** > **Open Browser with DevTools**:
-
-   ![Right-click > Open Browser with DevTools](./get-started-images/open-browser-with-devtools.png)
+*  Learn how to open and close DevTools by clicking the **Launch Instance** button, to experiment with the demo **Success** webpage.  See [Get started by clicking the Launch Instance button](./get-started-launch-instance.md).
```

Comment:
"Learn how to open and close DevTools" feels very repetitive. WE could replace it with "Use the extension by…"  as we already explained the task in the first sentence.